mautic/mautic
# Plugins page refactoring

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? |
| Deprecations? |

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
1. This PR contains refactored code related to shuffle.js integration.
2. It fixes issues with minimizing/maximizing sidebar on the plugins page.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Sometimes plugins doesn't "shuffle" correctly. Go to the plugins page, try to maximise or minimise the sidebar.

#### Steps to test this PR:
1. Go to the plugins page, try to maximise or minimise the sidebar. Plugins should be "repositioned" correctly.
